indico/modules/events/management/templates/settings.html

```diff
@@ -1,5 +1,6 @@
 {% extends 'events/management/base.html' %}
 {% from 'events/management/_settings.html' import render_event_settings %}
+{% from 'events/timetable/_draft_mode_warning.html' import render_draft_mode_warning %}
```

Comment:
Maybe this should be in contributions since that's where all the rest of the draft mode stuff resides

Reply:
Moved to `events/contributions/management`